src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java

```diff
@@ -0,0 +1,33 @@
+package com.github.dockerjava.api.command;
+
+import com.github.dockerjava.api.model.Service;
+
+import javax.annotation.CheckForNull;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * List containers
+ *
+ */
+public interface ListServicesCmd extends SyncDockerCmd<List<Service>> {
+
+    @CheckForNull
+    Map<String, List<String>> getFilters();
+
+    /**
+     * @param ids
+     *            - Show only services with the given ids
+     */
+    ListServicesCmd withIdFilter(String... ids);
+    /**
+     *
+     * @param names
+     *            - Show only services with the given names
+     */
+    ListServicesCmd withNameFilter(String... names);
```

Comment:
Good question, but if docker api accepts List(array), then it would make sense require the same type for method. I usually using `Collections.unmodifiableList(String)` when it possible. As i see
```java
    public static <T> List<T> singletonList(T o) {
        return new SingletonList<>(o);
    }
```
So we camke simple `asList(String)` to solve it
would look pretty fluent imho :)